packages/server/manage_container/server/libc_to_manage_container/L2m.go

```diff
@@ -345,21 +296,16 @@ func (s *server) GetDataList(ctx context.Context, in *pb.GetDataListRequest) (*p
 
 	errToken := s.authorize(token, []string{"demo", "dep"})
 	if errToken != nil {
-		return &pb.GetDataListResponse{
-			IsOk: false,
-		}, errToken
+		return nil, errToken
 	}
+
 	getDataList, err := s.m2dbclient.GetDataList()
 	if err != nil {
 		AppLogger.Error(err)
-		return &pb.GetDataListResponse{
-			IsOk:   false,
-			Result: "",
-		}, nil
+		return nil, errors.New("")
```

Comment:
Could we use a following code?  
From the client side, the contents of the error would not be understood.

```suggestion
		return nil, err
```

Reply:
[fdc75cf](https://github.com/acompany-develop/QuickMPC/pull/161/commits/fdc75cf15dfdb5a18c23e339f037f4bc7bd65f57)

Reply:
Thank you!  checked.